packages/in_app_purchase/lib/src/in_app_purchase/purchase_details.dart

```diff
@@ -137,6 +157,17 @@ class PurchaseDetails {
   /// This is null on Android.
   final PurchaseWrapper billingClientPurchase;
 
+  /// The developer has to call [InAppPurchaseConnection.completePurchase] if the value is `true`.
+  ///
```

Comment:
nit: Really the method should be called when this is true and the product has actually been delivered to the user. I think it's important to mention both.

Reply:
done